woocommerce/woocommerce
# Fix Marketplace theme labels and links

### Submission Review Guidelines:


### Changes proposed in this Pull Request:

This PR fixes a few details in the Marketplace section.
- Search placeholder text includes themes.
- Updated title for themes no result product list
- Open product links in the same page



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. [Open the Marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions)
2. Confirm the search placeholder is _Search for extensions and themes_
3. Search for a term that won't return anything
4. Confirm that the theme no result product list title is _Our favorite themes_
5. Click on a product and confirm it opened in the same tab



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Fix Marketplace theme labels and links.

</details>
